requirements.in

```diff
@@ -1,2 +1,3 @@
+# any dependency added here should also be added in pyproject.toml
 bo4e>=0.5.5
-# any dependency added here should also be added in pyproject.toml
\ No newline at end of file
+bomf>=0.6.8
```

Comment:
und wenn das hier ist, dann müsstest du es bitte auch in die pyproject.toml

Reply:
du meinst bei den Keywords hinzufügen?

Reply:
In der pyproject.toml gibts n Feld für dependencies, einfach hinzufügen

Reply:
https://github.com/Hochfrequenz/intermediate-bo4e-migration-models/pull/13/commits/1321a1cd0ed97b50173494341a79f703b1edfbd3